src/Nethermind/Nethermind.Consensus/Validators/BlockValidator.cs

```diff
@@ -337,9 +337,9 @@ protected virtual bool ValidateEip4844Fields(Block block, IReleaseSpec spec, out
 
         ulong blobGasUsed = BlobGasCalculator.CalculateBlobGas(blobsInBlock);
 
-        if (blobGasUsed > Eip4844Constants.MaxBlobGasPerBlock)
+        if (blobGasUsed > spec.GetMaxBlobGasPerBlock())
         {
-            error = BlockErrorMessages.BlobGasUsedAboveBlockLimit;
+            error = string.Format(BlockErrorMessages.BlobGasUsedAboveBlockLimit, spec.GetMaxBlobGasPerBlock(), blobsInBlock, blobGasUsed);
```

Comment:
avoid string.Format here, make `BlobGasUsedAboveBlockLimit` a method and pass parameters